src/frontend/src/features/home/components/MosaLoginPage/MosaLoginPage.scss

```
@@ -0,0 +1,421 @@
+// Theme colors - Calendars uses violet
+$app-color: #a78bfa;
+$app-gradient-end: #6d3fde;
+
+// Animations
+@keyframes fadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)